### PR TITLE
Import license texts using "svn export"

### PR DESCRIPTION
**Pull request by sschuberth**
Please have a look at the individual commit messages for the details.

---
This change is [Reviewable](https://reviewable.io/reviews/heremaps/oss-review-toolkit/1170)

### CODE REVIEW

File: spdx-utils/src/main/resources/exceptions/Autoconf-exception-3.0 [hunk 1]
```diff
@@ -2,7 +2,7 @@ AUTOCONF CONFIGURE SCRIPT EXCEPTION
 
 Version 3.0, 18 August 2009
 
-Copyright © 2009 Free Software Foundation, Inc. <http://fsf.org/>
+Copyright � 2009 Free Software Foundation, Inc. <http://fsf.org/>
```
Comment (tsteenbe) [inline spdx-utils/src/main/resources/exceptions/Autoconf-exception-3.0]: Introduces character encoding issue for (C)

File: spdx-utils/src/main/resources/licenses/AGPL-1.0 [hunk 2]
```diff
@@ -2,7 +2,7 @@ AFFERO GENERAL PUBLIC LICENSE
 
 Version 1, March 2002
 
-Copyright © 2002 Affero Inc.
+Copyright � 2002 Affero Inc.
```
Comment (tsteenbe) [inline spdx-utils/src/main/resources/licenses/AGPL-1.0]: Introduces character encoding issue for (C)

File: spdx-utils/src/main/resources/licenses/AGPL-1.0-only [hunk 3]
```diff
@@ -2,7 +2,7 @@ AFFERO GENERAL PUBLIC LICENSE
 
 Version 1, March 2002
 
-Copyright © 2002 Affero Inc.
+Copyright � 2002 Affero Inc.
```
Comment (tsteenbe) [inline spdx-utils/src/main/resources/licenses/AGPL-1.0-only]: Introduces character encoding issue for (C)

File: spdx-utils/src/main/resources/licenses/AGPL-1.0-or-later [hunk 4]
```diff
@@ -2,7 +2,7 @@ AFFERO GENERAL PUBLIC LICENSE
 
 Version 1, March 2002
 
-Copyright © 2002 Affero Inc.
+Copyright � 2002 Affero Inc.
```
Comment (tsteenbe) [inline spdx-utils/src/main/resources/licenses/AGPL-1.0-or-later]: Introduces character encoding issue for (C)

File: spdx-utils/src/main/resources/exceptions/Autoconf-exception-3.0 (hunk 1 above)
Comment (sschuberth) [reply spdx-utils/src/main/resources/exceptions/Autoconf-exception-3.0]: Thanks, turns out there indeed was an encoding issue.

**Pull request merged**